openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/targets/INST2/lib/sim_inst.py

```diff
@@ -238,7 +238,7 @@ def solar_panel_thread_method(self):
                 self.solar_panel_thread_cancel = False
                 break
 
-    def graceful_kill(self):
+    def graceful_kill(self, timeout):
```

Comment:
Adding a required argument to graceful_kill is super breaking.  This is an established pattern in both Ruby/Python - No arguments.